eng/Packages.props

```diff
@@ -1,4 +1,8 @@
 <Project>
+  <!--
+    Make sure to update the binding redirects (in src\MSBuild\app.config) for any changes to
```

Comment:
👍👍👍👍

Reply:
Updated to include the 64bit appconfig in that comment as well :)